### PR TITLE
Content/create whatsapp chatbot

### PR DESCRIPTION
**Pull request by younesbenallal**

### CODE REVIEW

File: apps/landing-page/content/create-whatsapp-chatbot.mdx [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+### Building Your Chatbot Flow
+
+Once you've created your account, it's time to start designing your chatbot's conversation flow. Typebot's visual builder makes this process intuitive and engaging.
+
+<Image
+  src="/images/blog/create-whatsapp-chatbot/typebot-blocs.png"
+  alt="All blocs offered by Typebot"
+  width={318}
+  height={535}
```
Comment (baptisteArno) [inline apps/landing-page/content/create-whatsapp-chatbot.mdx:25]: It should be spelled "block" :)

File: apps/landing-page/content/create-whatsapp-chatbot.mdx [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+<Image
+  src="/images/blog/create-whatsapp-chatbot/typebot-share-button.png"
+  alt="Typebot share button"
+  width={289}
+  height={49}
+/>
+
+To access the WhatsApp preview:
+
+1. Click on the "Preview" button in the Typebot editor.
```
Comment (baptisteArno) [inline apps/landing-page/content/create-whatsapp-chatbot.mdx:167]: The button was renamed "Test"

File: apps/landing-page/content/create-whatsapp-chatbot.mdx [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+
+### Contact Information and System Variables
+
+One powerful aspect of WhatsApp is automatically capturing user contact information, such as their name and phone number.
+
+Typebot makes it easy to leverage this data within your chatbot flow using dedicated system variables.
+
+For example, you can use the `{{waba.name}}` variable to personalize your chatbot's greetings and responses with the user's name. This simple touch can make the experience feel more human and tailored to the individual.
+
+Similarly, the `{{waba.phone}}` variable can pre-populate input fields or integrate with systems that require the user's phone number, streamlining the overall experience.
```
Comment (baptisteArno) [inline apps/landing-page/content/create-whatsapp-chatbot.mdx:264]: This is not how it works (see https://docs.typebot.io/deploy/whatsapp/overview#contact-information), the user need to add a set variable block with the WhatsApp values

**Review by baptisteArno — CHANGES_REQUESTED** [review]
Nice one overall! 🔥
If you can rename every "bloc" into "block", it would be great (also the file names)

**Review by baptisteArno — CHANGES_REQUESTED** [review]
Would it be possible to improve the images sharpness?

**Comment by younesbenallal** [comment]
Are images better now?

File: apps/landing-page/content/create-whatsapp-chatbot.mdx [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+During testing, pay close attention to:
+
+- **User Experience**: Ensure the conversation flow is natural, intuitive, and free from confusing transitions.
+- **Media Rendering**: Verify that images, videos, and other media assets display correctly.
+- **Button Functionality**: Test each button to ensure it triggers the appropriate response or action.
+- **Input Field Validation**: Check that input fields function correctly, validating user input and triggering appropriate conditional logic.
+- **Edge Cases**: Explore potential edge cases or unexpected user inputs to ensure graceful handling and appropriate responses.
+
+Document any issues or areas for improvement. This will help streamline debugging and optimization.
+
```
Comment (baptisteArno) [inline apps/landing-page/content/create-whatsapp-chatbot.mdx:188]: I think the entire `### Integrating Typebot with WhatsApp` section should be positioned here, after the testing phase. It makes more sense to me 🙏

**Review by baptisteArno — CHANGES_REQUESTED** [review]
We also have 2 broken images to fix 🙏

CleanShot 2024-05-23 at 11 25 16@2x

CleanShot 2024-05-23 at 11 25 09@2x

**Review by baptisteArno — APPROVED** [review]

**Pull request merged**